source/_integrations/history_stats.markdown

```diff
@@ -48,6 +48,10 @@ entity_id:
   description: The entity you want to track.
   required: true
   type: string
+unique_id:
+  description: An ID that uniquely identifies this entity. This allows changing the `name`, `icon` and `entity_id` from the web interface.
```

Comment:
```suggestion
  description: An ID that uniquely identifies this entity. Set this to a unique value to allow customization through the UI.
```

Let's go with something more generic since the UI may change over time and allow additional customizations which would make this stale.